### DISCUSSION TITLE
Whisper Const-me large-v3

### DISCUSSION THREAD

**Discussion by reinduhr** [SubtitleEdit/subtitleedit#9122] [Ideas]
Hello everyone,

I noticed Subtitle Edit is currently downloading Whisper V2 while V3 is out for over a year now. Are there plans to switch to this new version?
-Whisper Large V2 was released on December 8, 2022. 
-Whisper Large V3 was released in November 2023, on the OpenAI Dev Day.

Have a nice day! 😁

**Comment by niksedk** [comment]
Sorry, Const-me is not updated anymore :(

**Comment by darnn** [comment]
On the Const-me GitHub, there was someone who created a build which ostensibly supported large-v3, but the results were much worse than with other versions of Whisper and v3. Your best bet in Subtitle Edit itself is either CPP, CPP cuBLAS or Faster-Whisper-XXL. They all support v3. Try each for yourself and see which one works best for you in terms of speed and the quality of the output.
